doc/tor.md

```diff
@@ -1,4 +1,4 @@
-TOR SUPPORT IN BITCOIN
+TOR SUPPORT IN DOGECOIN
 ======================
```

Comment:
You're going to hate me - could you add an extra = to the end of the 2nd line so it underlines the entire text please?